superset/db_engine_specs/hive.py

```diff
@@ -299,16 +299,19 @@ def get_columns(
     @classmethod
     def where_latest_partition(cls, table_name, schema, database, qry, columns=None):
```

Comment:
This should be identical with what's in `BaseEngineSpec`: https://github.com/apache/incubator-superset/blob/master/superset/db_engine_specs/base.py#L565-L572